Cargo.toml

```diff
@@ -48,6 +50,5 @@ optional = true
 version = "0.1.4"
 optional = true
 
-[build-dependencies]
-phf_generator = "0.7.4"
-phf_shared = "0.7.4"
+[build-dependencies.string_cache_codegen]
+version = "0.2.11"
```

Comment:
> However, I believe that won't work for downloads from crates.io because of `exclude = ["string-cache-codegen/**/*"]`. I added this exclude line because I think string-cache should consume string-cache-codegen from crates.io (like anything else will need to).

I'm pretty sure you just need a `path` dependency, with no `exclude` at all, just like phf does.